e2e/specs/interactive-builder.spec.ts

```diff
@@ -224,6 +224,123 @@ test('Create a form using the interactive builder', async ({ page, context }) =>
     const editFormPageURL = page.url();
     formUuid = editFormPageURL.split('/').slice(-1)[0];
   });
+
+  await test.step('Then I click the `Edit` button on the form name I need to edit', async () => {
```

Comment:
It'll be better if we break this into a separate spec on its own like `edit-form-with-interactive-builder` (that starts similar to how you delete a form, as in edits an existing form) so that it can in parallel, and rename this current file into something like `create-form-with-interactive-builder.spec`